docs/schemas/README.md

```diff
@@ -0,0 +1,17 @@
+# Configuration Schemas
+
+This folder contains the schemas for the configuration used by the Microsoft.FeatureManagement library.
+
+# 1.0.0
+
+The [1.0.0 schema](./FeatureManagement.v1.0.0.json) is supported by Microsoft.FeatureManagement version 1.x - 3.x.
+
+* Allows feature flags to be defined.
+
+# 3.0.0
```

Comment:
Kinda odd to see we jump from 1.0.0 to 3.0.0. I don't think we should bother to align the schema version with the package version. I would even prefer we decouple it, so we can version the package as needed. I would suggest we version it as 2.0.0.

Reply:
That's fine with me.

Reply:
Cool, so you will rename the version in the next update?

Reply:
Done now. looks like those changes didn't get pushed in last update.